### ISSUE TITLE
Invisible split action bar reacts tapping

### ISSUE THREAD

**Issue by naoak** [JakeWharton/ActionBarSherlock#246]
Thanks this great libarary in advance.

I found the title issue in my phone (API 10). 

Split action bar can be visible but  still  handles tapping events
when the `hide()` method is called  in `onOptionsItemSelected()`,  not  in `onCreate()`.
If I post `hide()` in `onCreate()`, the issue occurs too.

This does not occur in API 15 emulator.

I think this issue is related to the following article.
http://stackoverflow.com/questions/4728908/android-view-with-view-gone-still-receives-ontouch-and-onclick

If I call the `clearAnimation()` method of the split view like below,  the issue doesn't occur. 

``` java
// ActionBarImpl.java
 if (mSplitView != null && mContextDisplayMode == CONTEXT_DISPLAY_SPLIT) {
    mSplitView.setVisibility(View.GONE);
    // added
    mSplitView.clearAnimation();
 }
```

But, this solution is not the best, I think.
Native action bar implementation seems to be the same as ABS's at this point.

**Comment by JakeWharton** [comment]
I had a fix in place for this but it clearly was not working. Just updated it to clear the animation on `GONE` so it should be fixed if you want to try the `4.0-wip` branch.

**Comment by naoak** [comment]
I've checked it works properly. Great!
